frontends/election-manager/src/utils/election.test.ts

```diff
@@ -16,3 +18,77 @@ test('getBallotPath allows digits in file names', () => {
     'election-d34db33f-precinct-north-springfield-id-21-style-77-English-live-absentee.pdf'
   );
 });
+
+describe('generateOvervoteBallot adds an overvote conflict when possible', () => {
+  const precinctId = electionSample.precincts[0].id;
+
+  test('minimally overvotes an initial candidate contest', () => {
+    const overvoteBallot = generateOvervoteBallot({
+      election: electionSample,
+      precinctId,
+    });
+    expect(overvoteBallot).toBeDefined();
+
+    const votes = overvoteBallot!.votes as VotesDict;
+    const overvoteContestId = electionSample.contests[0].id;
+
+    expect(Object.keys(votes)).toEqual([overvoteContestId]);
+    expect(votes[overvoteContestId]?.length).toEqual(2);
+  });
+
+  test('overvotes an initial yes-no contest', () => {
+    const election = _.cloneDeep(electionSample);
+
+    // remove all but the yes-no contests
+    _.remove(election.contests, (c) => c.type !== 'yesno');
```

Comment:
While I know we reached the consensus that lodash is fine for us to use in these tests, especially for `cloneDeep`, it's still nice to avoid using it when easily possible. This `remove` can be easily replaced with a `filter`:

```ts
election.contests = election.contests.filter(c => c.type === 'yesno')
```